public/app/features/alerting/unified/components/MoreButton.tsx

```diff
@@ -4,18 +4,21 @@ import { Button, ButtonProps, Icon, Stack } from '@grafana/ui';
 
 import { Trans, t } from '../../../../core/internationalization';
 
-const MoreButton = forwardRef(function MoreButton(props: ButtonProps, ref: Ref<HTMLButtonElement>) {
+const MoreButton = forwardRef(function MoreButton(
+  props: ButtonProps & { title?: string },
```

Comment:
There was the option to create a new component exactly like this but with different title. I decided to reuse this one, but I can follow the other approach.